docs/docs/guides/how-to-parameterize-cassandra.md

```diff
@@ -0,0 +1,396 @@
+
+To quickly demonstrate the latter capability, simply run:
+
+```bash
+kurtosis clean -a && kurtosis run --enclave cassandra-cluster github.com/kurtosis-tech/cassandra-package '{"num_nodes": 5}' 
+```
+
+Which should give you the same result as we saw when we ran our local `main.star` file with 5 nodes specified as an argument. However this time, we’re actually using a `main.star` file hosted [remotely on Github and running that instead][github-cass-package]!
+
+**Any Kurtosis environment definition can be converted into a runnable, shareable artifact that we call a Kurtosis Package.** What we just did was run a remotely-hosted Kurtosis package.
+
+While this guide won’t cover the steps needed to convert your Starlark file and export it for others to use as a Kurtosis Package, you can check out our docs to learn more about how to turn a Starlark file into a [runnable Kurtosis package][runnable-packages] using only a [`kurtosis.yml`][kurtosis-yml] file.
+
+### Review
+What we did with our most recent command was execute a remotely-hosted `main.star` from a Kurtosis package on [Github][github-cass-package]. That remotely-hosted `main.star` file has the same code as our local `main.star` file and, with only a [locator][locator], Kurtosis knew we were referencing a publicly-hosted runnable package. 
+
+The entirety of what we wrote locally in our `main.star` file can be packaged up and pushed to the internet (Github) for anyone to use to instantiate an n-node cassandra cluster by simply adding a [`kurtosis.yml`][kurtosis-yml] manifest to your directory and publishing the entire directory to Github. Read more about how to do this [here][runnable-packages].
+
+Turning your Starlark code into a runnable Kurtosis package and making it available on Github means any developer will be able to use our Kurtosis package as a building block in their own environment definition or to run different tests using various configurations of nodes in a Cassandra cluster.
+
+This behavior is bi-directional as well. Meaning, we can import any remotely hosted Kurtosis package and use it’s code inside our own Kurtosis package with the `import_module` instruction like so:
+
+```python
+lib = import_module("github.com/foo/bar/src/lib.star")
+
+def run(plan,args)
+lib.some_function()
+lib.some_variable()
+```
+
+**This feature we just witnessed is the composability property of environments created using Kurtosis and allows environments, and their components, to be easily composed and connected together without needing to know the inner workings of each setup.**
+
+Conclusion
+----------
+And that’s it! To recap this short guide, together we:
+1. Wrote an environment definition that instructs Kurtosis to set up a 3 node Cassandra cluster in an isolated environment called an Enclave (over Docker),
+2. We added a few lines of code to our `main.star` file to parametrize our environment definition to increase the flexibility and use cases with which it can be used, and,
```

Comment:
```suggestion
2. Added a few lines of code to our `main.star` file to parametrize your environment definition to increase the flexibility and use cases with which it can be used, and,
```